controller/src/types/modules.ts

```diff
@@ -73,6 +74,7 @@ export interface IPermissionManager<T = Record<string, BaseSubject<string>>> {
     createPermissions<K extends SubjectKey<T>>(resource: string, subject: T[K], permissions: Permission[]): Promise<void>
     // Does not update the index file
     editPermissions<K extends SubjectKey<T>>(resource: string, item: IndexItem, subject: T[K], permissions: Permission[]): Promise<void>
+    deletePermissions<K extends SubjectKey<T>>(resource: string, subject: T[K]): Promise<void>
```

Comment:
Where is this being called? Based on the diff I have the feeling you only update the index file but not the ACL?